pillar/metalk8s/kubernetes/kubelet.sls

```diff
@@ -0,0 +1,8 @@
+certificates:
+  kubeconfig:
+    files:
+      kubelet:
+        path: /etc/kubernetes/kubelet.conf
+        cn: "system:node:{{ grains.id }}"
```

Comment:
So, reading on, it appears this is used to ensure the CN in the cert matches what's expected. However, I guess the same should then be done for `o`.

Related, not sure at this place we should actually validate `cn` and `o`: this is coded in the state that creates the cert in the first place. The beacon shouldn't really care and only ensure lifetime is acceptable. It has (and should not have) knowledge about who this cert is for, and call the applicable state(s) to (re)generate it.

Reply:
Oh yes there is also an `O` (but hard-coded in the formula), the `CN` here is a leftover (we needed in `metalk8s_kubeconfig.validate` function before we made it optional), but maybe we could put everything in this dict, so we have only one place with everything.

Reply:
Let's keep this "extracting all info about certs into pillar" for a later time. Would make sense to remove the CN for now, since it's not used, and see later how we want to improve this.

Reply:
I've removed this from the pillar.